web/client/components/map/openlayers/Map.jsx

```diff
@@ -245,10 +246,12 @@ class OpenlayersMap extends React.Component {
             }
         }
 
-        if (!isEqual(newProps.bbox, this.props.bbox) && newProps.bbox && !isEmpty(newProps.bbox) && newProps.zoomToExtent) {
-            //
-            const extent = newProps && newProps.bbox && newProps.bbox.bounds;
-            let bounds = CoordinatesUtils.reprojectBbox(extent, newProps.bbox.crs, newProps.projection);
+        if (!isEqual(newProps.wpsBounds, this.props.wpsBounds) && newProps.wpsBounds && !isEmpty(newProps.wpsBounds) && newProps.zoomToExtent) {
```

Comment:
Maybe we can change the way a hook is registered by adding the id aswell with the name.

then we can retieve a specific hook by specifing the name + map id

this would require a refactor of
- utilities in MapUtils for registering and fetching hooks
- update any reference to those utilities
- Map components registerHook calls to be updated